### ISSUE TITLE
Extend CxG validator - to cover https://github.com/dosumis/single-cell-curation/tree/species_general (Damien & I will propose this to scFAIR shortly.

### ISSUE THREAD

**Issue by dosumis** [EBISPOT/scxa_kg#10]

**Comment by anitacaron** [comment]
There aren't [relevant commits](https://github.com/chanzuckerberg/single-cell-curation/compare/main...dosumis:single-cell-curation:species_general) in this branch; what is this supposed to have?

**Comment by dosumis** [comment]
@gouttegd - I think you edited more extensively than commits that can be found in https://github.com/dosumis/single-cell-curation/tree/Broadening_species_specificity  Do you have your own fork?

**Comment by gouttegd** [comment]
@dosumis @anitacaron No, I don’t have my own fork. We did all in this fork. My edits are in the [broadening-species-specificity](https://github.com/dosumis/single-cell-curation/tree/broadening-species-specificity) branch.

**Comment by gouttegd** [comment]
What we did was based on version 4.0.0 of the original schema (latest version at the time). We should probably update to at least 5.0.0 or 5.1.0.

**Comment by gouttegd** [comment]
> We should probably update to at least 5.0.0 or 5.1.0.

Just done that.

**Comment by dosumis** [comment]
Got it.  So the latest version of the file is here https://github.com/dosumis/single-cell-curation/blob/553cbfe0264dbac23215df20562edeff7a0628bf/schema/5.1.0-broad/schema.md

OK to share on scFAIR and ask about shifting hosting to their Org?

**Comment by gouttegd** [comment]
> OK to share on scFAIR and ask about shifting hosting to their Org?

Sure.
